admin/python/lsst/qserv/admin/dataLoader.py

```diff
@@ -582,8 +582,14 @@ def _loadChunkedData(self, database, table):
 
                 # make tables if needed
                 if chunkId not in self.createdChunks:
-                    wmgr.createChunk(database, table, chunkId, overlap=self.partOptions.isSubChunked)
-                    self.createdChunks.add(chunkId)
+                    try:
+                        wmgr.createChunk(database, table, chunkId, overlap=self.partOptions.isSubChunked)
+                        self.createdChunks.add(chunkId)
+                    except ServerError as exc:
+                        if exc.code == 409:
+                            self._log.info('Chunk %r exists for table %r', chunkId, table)
+                        else:
+                            self._log.critical('Failed to create chunk %r for table %r', chunkId, table)
```

Comment:
Actually - yes, I would like just to post an information message that the chunk already exists (because some other parallel process has created it) and continue as if noting bad has happened. Note that code 409 is reported when the table already exists!